### ISSUE TITLE
Fullchain Cert Generation, relative path assumption

### ISSUE THREAD

**Issue by FuzzySecurity** [outflanknl/RedELK#181]
This line in the install script assumes a relative path for `config.cnf`

https://github.com/outflanknl/RedELK/blob/master/elkserver/install-elkserver.sh#L437

In my case that was not correct, during deployment I use Ansible regex replace to put in the correct full path.

```yml
  - name: Fix beta v4 deployment path bug
    replace:
      path: /opt/RedELK-Install/install-elkserver.sh
      regexp: '\.\.\/certs\/config\.cnf'
      replace: '/opt/RedELK/build/RedELK-2.0.0-beta.4/certs/config.cnf'
```

**Comment by FuzzySecurity** [comment]
Just to clarify I replace it to use **my correct path**. But maybe there should be a generic fix for this where you supply the path to `config.cnf` when you run the script.

**Comment by fastlorenzo** [comment]
@FuzzySecurity maybe check the Ansible playbook I'm maintaining that installs RedELK server: https://github.com/fastlorenzo/redelk-server

Same for the client (both C2/redirs): https://github.com/fastlorenzo/redelk-client

I'll let @MarcOverIP decide if we need to put it as a variable in the install-elkserver.sh script

**Comment by MarcOverIP** [comment]
Not a big fan of introducing new variables. But I get the issue that in some installation flows the initial folder is not accessible anymore.

Would it make things easier if, as part of the initial-setup.sh installer, we would copy the certs/config.cnf to elkserver/initial-setup-data/certs/config.cnf?

**Comment by FuzzySecurity** [comment]
@MarcOverIP you mean copy `config.cnf` into the installer package? I think that would work yes.

@fastlorenzo I was updating our existing internal deployment it's all working now. I just wanted to highlight this because I think it's not intuitive to expect the file to be at a relative path.

**Comment by MarcOverIP** [comment]
Fixed in commit 435d2fe806d5461ec14d7537cd9e5cebc26840dd.

The file is copied to elkserver/initial-setup-data/certs/config.cnf. The elkserver installer references it from there. This should be good. If not, feel free to reopen the issue.
